src/components/Form.jsx

```diff
@@ -1,17 +1,23 @@
 import {useState} from "react";
 import {useId} from "react";
+import {useDispatch} from "react-redux";
+import {addtodo} from "../redux/todo/actions"
+import { v4 as uuid } from 'uuid';
 
 export function Form() {
     const [task, setTask] = useState('');
-
+    const dispatch = useDispatch();
     function handleSubmit(event) {
         event.preventDefault();
+        const unique_id = uuid();
+        const small_id = unique_id.slice(0,8)
```

Comment:
why are we slicing the unique Id? if there is no. specific reason then dont slice it